docs/format/db.rst

```diff
@@ -58,6 +71,29 @@ Related tools:
    * Ex: CLB is solved by first solving LUT bits, and then solving FF bits
 
 
+Interconnect PIP Tags
+^^^^^^^^^^^^^^^^^^^^^
+
+Tags for interconnect PIPs are stored in the `segbits_int_l.db` and `segbits_int_r.db` database files. For example, look at `segbits_int_l.db` for the bits that configure the `INT_L` tile in a `CLBLL_L` or `CLBLM_L` segment.
+
+Tags that enable interconnect PIPs have the following syntax: `<tile_type>.<destination_wire>.<source_wire>`.
+
+The `<tile_type>` may be `INT_L` or `INT_R`. The destination and source wires are wire names in that tile type. For example, consider the following entry in `segbits_int_l.db`: `INT_L.NL1BEG1.NN6END2 07_32 12_33`
```

Comment:
This is probably a good place to reference tileconn as the documentation that explains how wires are connected outside of switchboxes (which is what pips document).

Reply:
Added a FIXME.